root/static/scripts/common/MB/Control/Autocomplete.js

```diff
@@ -831,16 +838,18 @@ MB.Control.autocomplete_formatters = {
 
         if (comment.length)
         {
-            a.append(' <span class="autocomplete-comment">(' +
-                     _.escape(commaOnlyList(comment)) + ')</span>');
+            a.append(' <span class="autocomplete-comment">' +
+                     _.escape(bracketed(commaOnlyList(comment))) + '</span>');
         }
 
         var area = item.area;
         if (item.typeName || area) {
             a.append('<br /><span class="autocomplete-comment">' +
-                     (item.typeName ? _.escape(item.typeName) : '') +
+                     (item.typeName ? _.escape(lp_attributes(item.typeName, 'place_type')) : '') +
                      (item.typeName && item.area ? ', ' : '') +
-                     (area ? _.escape(area.name + ', ' + renderContainingAreas(area)) : '') +
+                     (area ? _.escape(area.name +
+                       (renderContainingAreas(area) ? ', ' + renderContainingAreas(area) : ''))
```

Comment:
`area.containment && area.containment.length` would be less expensive than calling the function twice